Lib/ufo2ft/util.py

```diff
@@ -34,13 +34,18 @@ def makeOfficialGlyphOrder(font, glyphOrder=None):
     If ".notdef" glyph is present in the font, force this to always be
     the first glyph (at index 0).
     """
+    reorderSpace = False
     if glyphOrder is None:
         glyphOrder = getattr(font, "glyphOrder", ())
+        reorderSpace = True
```

Comment:
from the POV of ufo2ft, an empty public.glyphOrder list meant the same as an absent one, in both cases all the glyphs (that are not part of the explicit list of ordered glyphs, because the list itself is empty or absent) were sorted alphabetically after the .notdef glyph.
So I think we don't need to distinguish the None-ness of glyphOrder from it being empty or incomplete (i.e. not listing "space" among its ordered glyphs).
You could revert to simply checking the font.glyphOrder property, since both libraries should return an empty list in the case it is not set.

Also here you could rephrase it like this

```
if glyphOrderf is None:
    glyphOrder = getattr(font, "glyphOrder", ())
reorderSpace = "space" in glyphOrder
```

Reply:
sorry for the typo above, of course I meant ... reorderSpace = "space" **not** in glyphOrder

Reply:
Makes sense.